Automattic/wc-calypso-bridge
# Re-enable Site visibility settings tab for free trial plans

### Changes proposed in this Pull Request:



Closes https://github.com/Automattic/wc-calypso-bridge/issues/1508

1. Re-enable Side visibility tab for free trial sites
2. Hide Site visibility form and display a notice with copy, `You're currently using a free trial! To get access to the full range of features, please upgrade to a paid plan.` with `paid plan` a link to `https://wordpress.com/plans/SITE_URL`


<img width="1147" alt="Screenshot 2024-09-13 at 09 44 01" src="https://github.com/user-attachments/assets/71143820-65a7-4200-8387-d444d7467451">








### How to test the changes in this Pull Request:




1. Setting up a free eCommerce trial site for development peapX7-1D4-p2
2. Enable LYS feature flag (via beta tester or you can remove [these lines](https://github.com/Automattic/wc-calypso-bridge/blob/507bde7b933c60bb2cb7efdfad66b7bd8581b39b/includes/class-wc-calypso-bridge-woocommerce-admin-features.php#L143-L147)).
3. Go to /wp-admin/admin.php?page=wc-settings&tab=site-visibility
4. Verify that the Site visibility form is hidden and the notice is displayed.
5. Click on the `paid plan` link and verify that it redirects to the plans page.




### Other information:




### FOR PR REVIEWER ONLY:
